src/renderer/transforms/forge.ts

```diff
@@ -15,13 +15,13 @@ export async function forgeTransform(files: Files): Promise<Files> {
       const parsed = JSON.parse(files.get(PACKAGE_NAME)!);
 
       // devDependencies
-      parsed.devDependencies = parsed.devDependencies || {};
-      parsed.devDependencies['@electron-forge/cli'] = '6.0.0-beta.52';
-      parsed.devDependencies['@electron-forge/maker-deb'] = '6.0.0-beta.52';
-      parsed.devDependencies['@electron-forge/maker-rpm'] = '6.0.0-beta.52';
-      parsed.devDependencies['@electron-forge/maker-squirrel'] =
-        '6.0.0-beta.52';
-      parsed.devDependencies['@electron-forge/maker-zip'] = '6.0.0-beta.52';
+      const forgeVersion = '6.0.0-beta.59';
```

Comment:
I have no idea whether this will work with parcel, but:

```suggestion
      const forgeVersion = require(path.join(__dirname, '../../../package.json')).devDependencies['@electron-forge/cli'];
```

Reply:
@malept I wanted to ask first, since the generated package.json pins to the exact release number, but electron-fiddle's package.json uses ^ notation on the version, and I don't know if the difference is an oversight or intentional.

Should we copy the version number straight over from electron-fiddle's package.json?

Reply:
I think the caret is fine, that's how `electron-forge init` works.

Reply:
OK. Tied to package.json's version in 5b6b8f5edbaae32fa516ee9f181555a9e95ef928